app/models/concerns/criminal_legal_aid_remuneration2020.rb

```diff
@@ -0,0 +1,7 @@
+module CriminalLegalAidRemuneration2020
+  extend ActiveSupport::Concern
+
+  def unused_materials_applicable?
+    case_type && ['Trial', 'Cracked Trial'].include?(case_type.name) && clar?
```

Comment:
don't rely on the name. rely on the `fee_type_code`. can this also be shortened with a safe navigator?
```suggestion
    ['GRTRL', 'GRRAK'].include?(case_type&.fee_type_code) && clar?
```